### ISSUE TITLE
Cannot switch to scalar_mono_polarized_double variant

### ISSUE THREAD

**Issue by leroyvn** [mitsuba-renderer/mitsuba2#112]
The `scalar_mono_polarized_double` variant builds correctly but cannot be imported in Python.

# System configuration

Platform: macOS 10.15.4
Compiler: Apple clang version 11.0.0 (clang-1100.0.33.17)
Python version: Python 3.7.6 :: Anaconda, Inc.
Mitsuba 2 version: 2694adf875c14b2a32ba6d7bbc26369daeead0fd
Compiled variants: "scalar_mono_polarized_double", "scalar_rgb"

# Problem

I cannot switch to the `scalar_mono_polarized_double`.

# Steps to reproduce

1. Compile Mitsuba with the aforementioned variants.
2. Run the following command:
   ```
   python -c "import mitsuba; mitsuba.set_variant('scalar_mono_polarized_double')"
   ```

The expected output is as follows:
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/vincent/Documents/src/thirdparty/mitsuba2/build/dist/python/mitsuba/__init__.py", line 162, in set_variant
    _import('mitsuba.core_' + value + '_ext')),
  File "/Users/vincent/miniconda3/envs/mitsuba2/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 670, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 583, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 1043, in create_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
ImportError: pybind11_type_alias(): source type not found!
```

**Comment by Speierers** [comment]
Thanks for reporting this @leroyvn !

This has definitely never been tested. It is probably a bug in `core/python/main_v.cpp`, I will have a look.

**Comment by Speierers** [comment]
Hi @leroyvn ,
Sorry for the delay. It should be fixed now: (b7897355)

Note that we have never really tested the polarized modes in double precision so you might run into other issues. Keep us posted on that 😉
